rubrik_cdm/cluster.py

```diff
@@ -39,6 +40,25 @@ def cluster_version(self, timeout=15):
             'cluster_version: Getting the software version of the Rubrik cluster.')
         return self.get('v1', '/cluster/me/version', timeout=timeout)['version']
 
+    def cluster_version_check(self, minimum_cluster_version, timeout=15):
+        """Determine if the Rubrik cluster is using running an earlier release than the provided CDM `minimum_cluster_version`.
+        If the CDM version is an earlier release than the "clusterVersion", the following message error message is thrown:
+            Error: The Rubrik cluster must be running CDM version {`minimum_cluster_version`} or later.
+
+        Arguments:
+            minimum_cluster_version {float} -- The minimum required version of Rubrik CDM you wish ensure is running.
+
+        Keyword Arguments:
+            timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {15})
+
+        Raises:
+            CDMVersionException -- When the Rubrik cluster is using running an earlier release than the provided CDM `minimum_cluster_version` the following error is raised:
+            The Rubrik cluster must be running CDM version {`minimum_cluster_version`} or later.
+        """
+
+        if float(self.cluster_version(timeout)[:3]) < float(minimum_cluster_version):
+            raise CDMVersionException(float(minimum_cluster_version))
```

Comment:
Misuse of exceptions. This method should be a boolean method that return false if the cluster is not running required version. Name of the method could be changed to reflect this better. The caller of this could raise an exception if it an exceptional state.

Reply:
I've gone back and worth on this way in terms of what should be returned (if anything). The original use case for this function was for creating functions that would only work with CDM version X so there was no need to check for true/false since it would be hard stop.

There are also use cases where we need to check for versions in-line to handle API changes in versions where it makes more sense to return a true/false. 

It's a toss up either way so I'm good with going with the best practice here and will return a true/false value

Reply:
renamed function to `minimum_installed_cdm_version` and updated return value.